packages/components/filters/src/filter-menu/filter-menu.tsx

```diff
@@ -93,14 +93,18 @@ export type TFilterMenuProps = {
    * controls whether menu is open on initial render
    */
   defaultOpen?: boolean;
+  /**
+   * controls whether menu is expanded
+   */
+  allowAutoMenuWidth?: boolean;
 };
 
-export const menuStyles = css`
+export const menuStyles = (props: Partial<TFilterMenuProps> = {}) => css`
   display: flex;
   flex-direction: column;
   align-items: flex-start;
   gap: ${designTokens.spacing30};
-  width: ${designTokens.constraint7};
+  width: ${props.allowAutoMenuWidth ? 'auto' : designTokens.constraint7};
```

Comment:
Using the `min-width` would have some weird effect on the menu when even in default cases, causing the width to be larger in menus with smaller(width) menu items .  Two options, either reduce the min-width to a smaller value or allow users set a value to change width to auto (`allowAutoMenuWidth`).
<img width="404" alt="image" src="https://github.com/user-attachments/assets/4cf27c7e-a99c-4d31-856f-40efca248d6d" />